app/views/dashboard/apps/appId/layout/appIdLayoutView.tsx

```diff
@@ -68,6 +70,12 @@ export default function AppIdLayoutView({
   const { flags } = useFeatureFlags()
   const [showSuccessModal, setShowSuccessModel] = useState<boolean>(false)
   const [showErrorModal, setShowErrorModel] = useState<boolean>(false)
+
+  const createdAtMinsDiffToday = useMemo(
+    () => dayjs().diff(endpoint?.createdAt, "minutes"),
```

Comment:
should this be "minute"?

![image](https://user-images.githubusercontent.com/10237065/191859215-4be7dc31-121b-4538-9204-fda70b4cb9e8.png)